patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -1063,6 +1078,19 @@
  				}
  
  				if (armor[k].type == 371) {
+@@ -8522,9 +_,9 @@
+ 					rangedDamage += 0.02f;
+ 					magicDamage += 0.02f;
+ 					minionDamage += 0.02f;
+-					magicCrit++;
+-					meleeCrit++;
+-					rangedCrit++;
++					magicCrit += 1;
++					meleeCrit += 1;
++					rangedCrit += 1;
```

Comment:
sad

Reply:
Indeed, even definition of Modifier + int operator doesn't solve this.